map/map.js

```diff
@@ -93,6 +93,18 @@ var DefineMap = Construct.extend("DefineMap",{
 					this.constructor.seal
 				);
 			});
+
+			var _computedGetter = Object.getOwnPropertyDescriptor(prototype, "_computed").get;
```

Comment:
`define-lazy-value` sets up a setter also. Does that need to be copied over?

Reply:
I suppose possibly, although is anything ever going to set `_computed` ?

Reply:
Probably not. I'm not sure.